### DISCUSSION TITLE
Anti Tab Complete/ plugin hider

### DISCUSSION THREAD

**Discussion by Insprill** [Zrips/CMI#2695] [Feature Requests]
**Description of feature/improvement request:**
Add a feature to block specific tab completions and block /pl and /plugins etc
---
### Detailed explanation of what it should do, when and in what situation: ###
Ok so basically add a anti tab complete feature that would have its own file and in it have the ability to block all tab completions except ones specified in a list unless the player has a special permission. And have groups so say default gets 1 set of tab completions they can do and another group get default + another list of commands.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Because all the other ones out there dont really work.. at all. and being able to block tab completions and your plugin list is a very important feature so 1 players only see what they can do and 2 so they can't copy your server.
 Hope this makes sense and can get implemented :)

**Comment by Va1a** [comment]
In the spigot.yml, there's a "commands.tab-complete" node set to 0. Set it to -1 and it should disable this. Although this wont work on a per group basis, it should be enough to hold you over until this feature is added. You can also negate the bukkit.command.help permission and that might work on a per group basis.
